multiver/bungee/propertynew/src/main/java/net/skinsrestorer/bungee/SkinApplierBungeeNew.java

```diff
@@ -56,13 +51,14 @@ protected void applyToHandler(InitialHandler handler, IProperty textures) {
     }
 
     @Override
-    public List<IProperty> getProperties(ProxiedPlayer player) {
+    public List<SkinProperty> getProperties(ProxiedPlayer player) {
         Property[] props = ((InitialHandler) player.getPendingConnection()).getLoginProfile().getProperties();
 
         if (props == null) {
-            return null;
+            return Collections.emptyList();
         }
 
-        return Arrays.stream(props).map(BungeePropertyNew::new).collect(Collectors.toList());
+        return Arrays.stream(props).filter(property -> property.getName().equals(SkinProperty.TEXTURES_NAME))
```

Comment:
Is this filter needed? Is this always the same on every other implementation?

Reply:
yeah because the property class is different on every other implementation